### DISCUSSION TITLE
New update

### DISCUSSION THREAD

**Discussion by aacentral** [code4recovery/12-step-meeting-list#414] [Q&A] [answered]
After update, Now my 12 step meeting list doesn’t work on the website but it still works on the meeting guide app. Any suggestions on how to fix it?

**Comment by brianw-area24** [comment, marked as answer]
My guess is this can be solved by going to the WordPress dashboard, select Settings -> Permalinks, and click on the Save Changes button. You don't need to change anything on the permalinks page, just "Save Changes"

If this doesn't work, go edit any meeting listing, and click Update. This should rebuild an internal cache used by the 12-step-meeting-list (TSML) plugin.

If neither of these suggestions work, please let us know, and we'll investigate farther.

**Comment by aacentral** [comment]
HI, 
Thank you so much!! the 2nd suggestion worked.


Inland Empire Central Office of Alcoholics Anonymous

| 
| 
| 
|  |  |



 |

 |
| 
|  | 
Inland Empire Central Office of Alcoholics Anonymous
 |

 |

 |


In Service,Carolyn ValencikExecutive Director909-825-4700P.O. Box 189897 Via Lata Suite A.
Colton Ca. 92324
https://aainlandempire.org/
 

    On Tuesday, July 6, 2021, 09:31:34 PM PDT, brianw-area24 ***@***.***> wrote:  
 
 


My guess is this can be solved by going to the WordPress dashboard, select Settings -> Permalinks, and click on the Save Changes button. You don't need to change anything on the permalinks page, just "Save Changes"

If this doesn't work, go edit any meeting listing, and click Update. This should rebuild an internal cache used by the 12-step-meeting-list (TSML) plugin.

If neither of these suggestions work, please let us know, and we'll investigate farther.
